static/src/stylesheets/module/site-messages/_engagement-banner.scss

```diff
@@ -221,8 +221,8 @@
 }
 
 // Override the css that is added to the container when an ad takeover happens
-.remote-banner {
-    width: auto !important;
+.site-message--banner.remote-banner {
```

Comment:
do we still need the `!important`s now that you have increased the specificity ?

Reply:
some of the style is added directly to the element, so I think the answer is yes:
<img width="254" alt="Screen Shot 2020-06-29 at 16 40 35" src="https://user-images.githubusercontent.com/1513454/86026408-4c9bef00-ba27-11ea-9d8e-6f06a131e480.png">

Reply:
Sadly, yes, because the ad script also injects CSS at the top of the page with `!important`.